Formula/e2fsprogs.rb

```diff
@@ -30,11 +37,14 @@ class E2fsprogs < Formula
   keg_only "this installs several executables which shadow macOS system commands"
 
   depends_on "pkg-config" => :build
-  depends_on "gettext"
 
-  # Remove `-flat_namespace` flag and fix M1 shared library build.
-  # Sent via email to theodore.tso@gmail.com
-  patch :DATA
+  on_macos do
+    depends_on "gettext"
+  end
+
+  on_linux do
+    depends_on "util-linux"
+  end
```

Comment:
I don't think `gettext` is needed for Linux build since upstream includes `*.gmo` files in both git (HEAD) and stable tarball. Everything else should be handled by glibc.

May want to rebase after #113300 to make sure `gettext` isn't in dependency tree due to `util-linux`.